ci/integration-tests-linux.groovy

```diff
@@ -1,68 +1,114 @@
 #!/usr/bin/env groovy
 
-pipeline {
-  agent none
+def credentials = [
+        [$class           : 'AmazonWebServicesCredentialsBinding',
+         credentialsId    : 'a20fbd60-2528-4e00-9175-ebe2287906cf',
+         accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+         secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'],
+        [$class       : 'FileBinding',
+         credentialsId: '23743034-1ac4-49f7-b2e6-a661aee2d11b',
+         variable     : 'CLI_TEST_SSH_KEY_PATH'],
+        [$class       : 'StringBinding',
+         credentialsId: '0b513aad-e0e0-4a82-95f4-309a80a02ff9',
+         variable     : 'DCOS_TEST_INSTALLER_URL'],
+        [$class       : 'StringBinding',
+         credentialsId: 'ca159ad3-7323-4564-818c-46a8f03e1389',
+         variable     : 'DCOS_TEST_LICENSE'],
+        [$class          : 'UsernamePasswordMultiBinding',
+         credentialsId   : '323df884-742b-4099-b8b7-d764e5eb9674',
+         usernameVariable: 'DCOS_USERNAME',
+         passwordVariable: 'DCOS_PASSWORD']
+]
 
-  options {
-    timeout(time: 6, unit: 'HOURS')
-  }
+def master_ip = 'UNKNOWN'
+def os = 'linux'
 
-  stages {
-    stage("Build Go binary") {
-      agent { label 'mesos-ubuntu' }
+pipeline {
+    agent { label 'mesos' }
```

Comment:
Previously we were building the binary from `mesos-ubuntu`, why switching to `mesos`? IIRC `mesos` relies on an alpine container, and building from there was creating binaries which wouldn't run on unbuntu.

We should run integration tests with a binary built the same way than the one we release.

Reply:
I think it was because the binary was dynamically linked and the glibc / musl differences.

Reply:
I switched to `mesos` from `mesos-ubuntu` since we don't need the whole ubuntu software to build binary and can rely on lightweight system.

Reply:
Ideal situation will be to use released version for integration tests. In fact we can have everything in a single pipeline. Right now we have binary build stage in 4 places, and E2E tests for different platforms shares 99% of code.

Reply:
As discussed, even though we use an alpine node it eventually runs in an Ubuntu container, so that's similar to the released one.